### ISSUE TITLE
No such package '@com_google_javascript_closure_library//closure/goog/soy'

### ISSUE THREAD

**Issue by lemito** [bazelbuild/rules_closure#572]
Hello,
I met following error:

ERROR: /Users/jacob/projects/feedbacker/src/assets/test/BUILD:32:1: no such package 'closure/templates': BUILD file not found on package path and referenced by '//:hello_soy'.
ERROR: Loading failed; build aborted.

`ERROR: /home/lemito/cpp-starter-kit/cpp/sources/clo/BUILD:35:19: no such package '@com_google_javascript_closure_library//closure/goog/soy': BUILD file not found in directory 'closure/goog/soy' of external repository @com_google_javascript_closure_library. Add a BUILD file to a directory to mark it as a package. and referenced by '//sources/clo:greeter_lib'
ERROR: Analysis of target '//sources/clo:greeter_lib' failed; build aborted: Analysis failed`

My files:
https://github.com/lemito/my-monorepo

**Comment by gkdn** [comment]
Same examples works in this repository so seems related to your repository setup. 
You are using a different com_google_javascript_closure_library version, it might be related to that.

**Comment by lemito** [comment]
Hello again :)

And how do I know which version of com_google_javascript_closure_library I have and which one I need to use for rules_closure to work properly ?

**Comment by gkdn** [comment]
If you don't override it, you should be getting [v20220301](https://github.com/bazelbuild/rules_closure/blob/7860dec6f1d7870d720ea6ef3f599c0397866fc9/closure/repositories.bzl#L718) at the moment.

**Comment by lemito** [comment]
@roman-kashitsyn Yes, it would be nice to either update the documentation or the rules_closure itself . Or will it itself no longer get updates since google itself stopped using it and moved to frameworks like angular?

**Comment by gkdn** [comment]
Closing this since closure library and goog/soy is [archived](https://github.com/google/closure-library).
